code/drasil-code/lib/Language/Drasil/CodeSpec.hs

```diff
@@ -47,6 +47,10 @@ data CodeSpec where
   purpose :: Purpose,
   -- | Example Background.
   background :: Background,
+  -- | Example Scope.
```

Comment:
I don't think this information should be in `CodeSpec` either. It is fine to have it in `SystemInformation` though.